lib/node_modules/@stdlib/blas/ext/base/sindex-of/docs/types/index.d.ts

```diff
@@ -0,0 +1,48 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+/**
+* Returns the index of a specified search element in a single-precision floating-point strided array.
+*
+* ## Notes
+*
+* -   If unable to find an element which equals a provided search element, the function returns `-1`.
+* -   If `offsetX` is less than zero, the starting index is resolved relative to the last array element, with the last array element corresponding to `offsetX = -1`.
```

Comment:
Offsets are never negative. Why are you designing the API this way? Please show me what BLAS package supports this behavior.